addon/index.js

```diff
@@ -417,6 +461,25 @@ export function changeset(obj, validateFn = defaultValidatorFn, validationMap =
       return this;
     },
 
+    /**
+     * Executes the changeset and saves the underlying content.
+     *
+     * @private
+     * @param  {Array} [keys=[]] Array of keys to push to
+     * @param  {String} [scope=''] A sring that can be prepended to the key.
+     * @return {Array}
+     */
+    _allKeys(object, keys = [], scope = '') {
```

Comment:
Hmm, will `Object.entries` cause any IE issues?

Reply:
umm.. Doesn't babel still pick that up?

Reply:
It seems like Babel needs a plugin for this - the local, transpiled version still has `Object.entries` in the code.

Reply:
:(

Reply:
done